app/src/main/java/com/example/androidfundamental/fragments/MyDialogFragment.kt

```diff
@@ -0,0 +1,51 @@
+package com.example.androidfundamental.fragments
+
+import android.os.Bundle
+import android.view.*
+import android.widget.EditText
+import android.widget.TextView
+import android.widget.TextView.OnEditorActionListener
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.Fragment
+import com.example.androidfundamental.R
+
+
+/**
+ * A simple [Fragment] subclass.
+ */
+class MyDialogFragment  // Empty constructor required for DialogFragment
+    : DialogFragment(), OnEditorActionListener {
+    private var mEditText: EditText? = null
```

Comment:
if this will be initialized later you could set like:

private lateinit var mEditText: EditText

In that way you avoid the mEditText!!